src/main/java/teammates/ui/controller/InstructorEditStudentFeedbackPageAction.java

```diff
@@ -80,14 +78,14 @@ private boolean hideQuestionsWithAnonymousResponses(FeedbackSessionQuestionsBund
             boolean isRecipientVisibleToInstructor = question.showRecipientNameTo.contains(FeedbackParticipantType.INSTRUCTORS);
             boolean isResponseVisibleToInstructor = question.showResponsesTo.contains(FeedbackParticipantType.INSTRUCTORS);
 
-            if (!isGiverVisibleToInstructor || !isRecipientVisibleToInstructor || !isResponseVisibleToInstructor) {
+            if (!isResponseVisibleToInstructor || !isGiverVisibleToInstructor || !isRecipientVisibleToInstructor) {
```

Comment:
Same as below, to avoid short-circuiting we check isResponseVisibleToInstructor first to achieve full branch coverage